get_goes_x.py

```diff
@@ -31,33 +33,32 @@
     print(('Warning: failed to open URL {}: {}'.format(url, last_err)))
     sys.exit(0)
 
-
-colnames = 'year month dom  hhmm  mjd secs short long ratio'.split()
-data_colnames = colnames[-3:]
-
 try:
-    dat = ascii.read(urldat, guess=False, Reader=ascii.NoHeader,
-                     data_start=3, names=colnames)
+    dat = Table(json.loads(urldat))
 except Exception as err:
     print(('Warning: malformed GOES_X data so table read failed: {}'
           .format(err)))
     sys.exit(0)
 
-# Strip up to two rows at the end if any values are bad (i.e. negative)
-for _ in range(2):
-    if any(dat[name][-1] < 0 for name in data_colnames):
-        dat = dat[:-1]
-
-mjd = dat['mjd'] + dat['secs'] / 86400.
+# Select only the GOES 16 data (which is all there is)
+dat = dat[dat['satellite'] == 16]
+# Make a table for each of the two wavelengths
+shortdat = dat[dat['energy'] == '0.05-0.4nm']['flux', 'time_tag']
+shortdat.rename_column('flux', 'short')
+longdat = dat[dat['energy'] == '0.1-0.8nm']['flux', 'time_tag']
+longdat.rename_column('flux', 'long')
+# Join them on time (seems to be OK for these data)
+joindat = join(shortdat, longdat)
 
-secs = DateTime(mjd, format='mjd').secs
+# Add a time column with Chandra secs
+secs = []
```

Comment:
I think this can be a one-liner:
```
joindat['time'] = Time(joindat['time_tag']).cxcsec
```
Astropy time understands zulu time and CXC seconds.

Reply:
Cool.  Thanks.